opensrp-reveal/src/main/java/org/smartregister/reveal/model/OfflineMapModel.java

```diff
@@ -0,0 +1,68 @@
+package org.smartregister.reveal.model;
+
+import com.mapbox.mapboxsdk.offline.OfflineRegion;
+
+import org.smartregister.domain.Location;
+
+import java.util.Date;
+
+public class OfflineMapModel {
+
+    private Location location;
+
+    private OfflineRegion offlineRegion;
+
+    private Date dateCreated;
+
+    public enum OFFLINE_MAP_STATUS {
+        READY,
+        DOWNLOAD_STARTED,
+        DOWNLOADED
+    }
+
+    public OfflineMapModel() {
+        setOfflineMapStatus(OFFLINE_MAP_STATUS.READY);
+    }
+
+    private OFFLINE_MAP_STATUS offlineMapStatus;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)